docs/sigrid-api-documentation.md

```diff
@@ -0,0 +1,154 @@
+Sigrid API documentation
+========================
+
+In addition to [Sigrid CI](../README.md), Sigrid also provides a more general-purpose REST API that you can use to obtain analysis results from Sigrid. This allows you to integrate data from Sigrid into your workflow. 
+
+## General usage
+
+- The Sigrid API base URL is `https://sigrid-says.com/rest/analysis-results/api/v1`. 
+- Authentication for the Sigrid API uses the same [authentication tokens](authentication-tokens.md) that are used by Sigrid CI. Your token's permissions are based on your user account, so the token can access the same systems that you can. 
+- You need to pass the authentication token to each request in the HTTP header `Authorization: Bearer {SIGRID_CI_TOKEN}`.
+- All end points will return HTTP status 401 if the token is invalid, or if the token is not authorized to access the portfolio and/or system.
+- All end points return JSON and therefore return a Content-Type of `application/json`.
+
+The following example shows how to call the Sigrid API using `curl`:
+
+```
+curl -D -  -H 'Accept: application/json' -H 'Authorization: {SIGRID_CI_TOKEN}' https://sigrid-says.com/rest/analysis-results/api/v1/maintainability/{customer}
+```
+
+In the example, `{customer}` refers to your Sigrid account name, and `{SIGRID_CI_TOKEN}` refers to your authentication token.
+
+## Available end points
+
+### Maintainability ratings
+
+`GET https://sigrid-says.com/rest/analysis-results/api/v1/maintainability/{customer}`
+
+Returns the maintainability ratings for all systems in your portfolio. The parameter `{customer}` refers to your Sigrid account name. 
+
+Example response:
+
+```
+{
+    "name": "my-sigrid-account-name",
+    "systems": [
+        {
+            "name": "my-system-name",
+            "clientName": "my-sigrid-account-name",
+            "maintainability": 5.24,
+            "maintainabilityDate": "2022-02-08",
+            "allRatings": [
+                {
+                    "maintainability": 4.96,
+                    "maintainabilityDate": "2022-02-06"
+                }
+            ]
+        }
+    ]
+}
+```
+
+The top-level `maintainability` and `maintainabilityDate` refer to the *current* state of each system. The `allRatings` array contains a list of all *historic* measurements, which can be used for reporting or trend information.
```

Comment:
This paragraph suggests actual responses contain more information than in the example above, which is a complete and syntactically correct JSON object. Suggestion: add `...` at places where a real response would contain more data.